simapp/test_helpers.go

```diff
@@ -69,6 +71,29 @@ func SetupWithGenesisAccounts(genAccs genaccounts.GenesisAccounts) *SimApp {
 	return app
 }
 
+// AddTestAddrs constructs and returns accNum amount of accounts with an
+// initial balanc of accAmt
+func AddTestAddrs(app *SimApp, ctx sdk.Context, accNum int, accAmt sdk.Int) []sdk.AccAddress {
+	testAddrs := make([]sdk.AccAddress, accNum)
+	for i := 0; i < accNum; i++ {
+		pk := ed25519.GenPrivKey().PubKey()
+		testAddrs[i] = sdk.AccAddress(pk.Address())
+	}
+
+	initCoins := sdk.NewCoins(sdk.NewCoin(app.StakingKeeper.BondDenom(ctx), accAmt))
+	totalSupply := sdk.NewCoins(sdk.NewCoin(app.StakingKeeper.BondDenom(ctx), accAmt.MulRaw(int64(len(testAddrs)))))
+	app.SupplyKeeper.SetSupply(ctx, supply.NewSupply(totalSupply))
```

Comment:
Good catch, will add it to the total supply. It can be easy to forget to update supply when writing tests so I think keeping this functionality encapsulated in this helper function will prevent useless debugging